symfony/client/src/core/util/composable/usePaginate.ts

```diff
@@ -74,7 +74,7 @@ function getPageParams(pageSize: number, currentPage: number) {
 function getQueryParams(query: any) {
   const params = JSON.parse(JSON.stringify(unref(query)));
   for (const [key, value] of Object.entries(params)) {
-    params[key] = value ? value : undefined;
+    params[key] = value === null || value === '' ? undefined : value;
```

Comment:
Hope you checked this working few other places as well?

Reply:
done